lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -329,7 +331,7 @@ static void fwdPoolMax(Tensor *inW, Tensor *outW, Handle<size_t> *SXY,
             }
           }
 
-          assert(!first && "Max value is uninitialized");
+          // assert(!first && "Max value is uninitialized");
```

Comment:
Why is this assertion commented out now?

Reply:
I see.  Please delete the assert then (rather than leaving it commented).

I guess it still seems like a mistake if someone chooses filter/pad such that it pulls a bunch of zeros into the output activation... but probably a user error rather than an assert condition.